Pre-merge build failure is unrelated:

```
/var/lib/buildkite-agent/builds/linux-56-59b8f5d88-dqhhn-1/llvm-project/github-pull-requests/llvm/lib/Target/AMDGPU/SIInstrInfo.cpp:9796:12: error: no matching function for call to 'has_single_bit'
    assert(llvm::has_single_bit(Mask) && "Invalid mask.");
```

--- a/compiler-rt/lib/rtsan/rtsan_diagnostics.cpp
+++ b/compiler-rt/lib/rtsan/rtsan_diagnostics.cpp
@@ -48,7 +48,7 @@ static void PrintStackTrace(uptr pc, uptr bp) {
 
 static void PrintError(const Decorator &decorator,
                        const DiagnosticsInfo &info) {
-  const auto error_type_str = [&info]() -> const char * {
+  const auto ErrorTypeStr = [&info]() -> const char * {
     switch (info.type) {
     case DiagnosticsInfoType::InterceptedCall:
       return "unsafe-library-call";
@@ -59,7 +59,7 @@ static void PrintError(const Decorator &decorator,
   };
 
   Printf("%s", decorator.Error());
-  Report("ERROR: RealtimeSanitizer: %s\n", error_type_str());
+  Report("ERROR: RealtimeSanitizer: %s\n", ErrorTypeStr());
 }
 
 static void PrintReason(const Decorator &decorator,

